packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -22,9 +22,9 @@ import type { QueryOption as QueryOptionType } from '../constants/query-option-d
 const editorStyles = css({
   width: '100%',
   minWidth: spacing[7],
-  // To match ace editor with leafygreen inputs
-  paddingTop: '5px',
-  paddingBottom: '5px',
+  // To match ace editor with leafygreen inputs.
+  paddingTop: '4.5px',
+  paddingBottom: '4.5px',
```

Comment:
You'll start getting half pixel rendering that looks bad on screens that are not retina, feel free to keep it, but I would say that 1px difference here is less annoying than introducing half pixels

Reply:
Looking closer I think this line I added to the default editor doesn't work:

https://github.com/mongodb-js/compass/blob/38ad2219a32b50664b382ab14ba5ff8142833bb8/packages/compass-editor/src/editor.tsx#L37

I see that the line height that ace sets is still set to 17px when it renders. Can you keep this change out? I would prefer to get to the root instead of working around this

Reply:
Moved out, probably best in another pr since that would impact other editors now. Thanks for catching.

Reply:
Thank you! I'll look into it first thing next week